renderer.js

```diff
@@ -53,12 +53,12 @@ class CanvasManager {
     alreadyAnimatingTo(new_transform) {
         if (this.animation_target) {
             let result = true;
-            result &&= this.animation_target.a == new_transform.a;
-            result &&= this.animation_target.b == new_transform.b;
-            result &&= this.animation_target.c == new_transform.c;
-            result &&= this.animation_target.d == new_transform.d;
-            result &&= this.animation_target.e == new_transform.e;
-            result &&= this.animation_target.f == new_transform.f;
+            if (result) result = this.animation_target.a == new_transform.a;
+            if (result) result = this.animation_target.b == new_transform.b;
+            if (result) result = this.animation_target.c == new_transform.c;
+            if (result) result = this.animation_target.d == new_transform.d;
+            if (result) result = this.animation_target.e == new_transform.e;
+            if (result) result = this.animation_target.f == new_transform.f;
```

Comment:
what happened here? Shouldn't this be `&=`?

Reply:
I'm not entirely sure what the reasoning behind the original code is here, however, this was changed because `&&=` doesn't seem to be supported in electron, which means that in turn it causes the script to crash inside of VSCode. This operator is unfamiliar to me, but according to the [Mozilla docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_AND_assignment), it "assigns if the left side is truthy", so this code should behave identically.

Reply:
sure, could you try `&=` though? if not, we can keep it as is

Reply:
This works, thanks